apps/satsale/docker-compose.yml

```
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   satsale:
-    image: satsale/satsale:0.1.11@sha256:d9b0cd4beb711f2e95562cdc76b05d71a10edb09c97920a3e21c861398320447
+    image: satsale/satsale:0.1.13@sha256:8d3889357e53cd3a8ec18e36d8dcbc107ac019cb59fd185cf07896dda3b7378d
```

This is the correct hash:

```suggestion
    image: satsale/satsale:0.1.13@sha256:98380ad6c5166d83e627c4d1f4efdbcfd5b9217096f7fb26b287fda89eebccbb
```

Super easy way to check is just with `docker pull`:

```
$ docker pull satsale/satsale:0.1.13
0.1.13: Pulling from satsale/satsale
Digest: sha256:98380ad6c5166d83e627c4d1f4efdbcfd5b9217096f7fb26b287fda89eebccbb
Status: Image is up to date for satsale/satsale:0.1.13
docker.io/satsale/satsale:0.1.13
```